packages/playground/src/styles.css

```diff
@@ -1,3 +1,53 @@
 @import '@harnessio/canary/src/styles.css';
 @import '@harnessio/unified-pipeline/dist/style.css';
 @import '@harnessio/unified-pipeline/src/styles.css';
+
+@import 'highlight.js/styles/github.css';
+/* @import '@git-diff-view/react/tailwind.css'; */
+
+.diff-style-root tr {
+  content-visibility: auto;
+}
+
+.diff-widget-tooltip {
+  position: relative;
+}
+
+.diff-widget-tooltip::after {
+  display: none;
+  box-sizing: border-box;
+  background-color: #555555;
```

Comment:
Add as a css variable and then use it here

Reply:
done